services/src/kilda-core/kilda-model/src/main/java/org/openkilda/model/KildaConfiguration.java

```diff
@@ -0,0 +1,59 @@
+/* Copyright 2019 Telstra Open Source
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.openkilda.model;
+
+import lombok.AccessLevel;
+import lombok.Builder;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.neo4j.ogm.annotation.GeneratedValue;
+import org.neo4j.ogm.annotation.Id;
+import org.neo4j.ogm.annotation.NodeEntity;
+import org.neo4j.ogm.annotation.Property;
+import org.neo4j.ogm.annotation.typeconversion.Convert;
+
+@Data
+@NoArgsConstructor
+@EqualsAndHashCode(exclude = "entityId")
+@NodeEntity(label = "kilda_configuration")
```

Comment:
This overlaps by naming with existing "config" node, which is used for feature toggles only as for now. Should we merge them or rename "config" to more specific "kilda_feature_toggles"?

Reply:
I think in the future we should rename "config" to "kilda_feature_toggles". Now it is not obvious that the "config" node contains feature toggles.

Reply:
Agreed with @timofei-durakov to handle this in #2441